README.md

```diff
@@ -791,44 +794,15 @@ Mining and inference mirrors The configuration file templates for `industryessen
 
 After successful training, the system will output the ID of the model. The user can find the corresponding file according to this id at `--model-location`. In fact, it is a tar file that can be extracted directly using the tar command to get the "mxnet" model file in parameters and JSON format.
 
-**How to view and modify the label file of the category name?**
-
-This file is in the package installation directory. Use this command to view the content of this file:
-
-```
-cat `pip show ymir-proto | grep 'Location: ' | cut -d ' ' -f2`/ymir/ids/type_id_names.csv
-```
-
-And you may get output like this:
-
-```
-0,frisbee
-1,car
-2,person
-3,surfboard
-4,cat
-5,bed
-6,clock
-7,pizza,pizza pie
-8,skateboard
-9,dining table,diningtable,board
-```
-
-Each line of this configuration file is separated by commas. The first item is the category id, followed by the category name.
-
-Pay attention to this line with id 7. It has 3 items: "7, pizza, and pizza pie". The first one is category id, the second one is a category name, and the third one is category alias.
-
-In the filter command, the -c and -C arguments are mapped to category ids with this file. All category aliases have the same meaning as category names.
-
-You can edit this file by vi and other text editing tools. You can add alias to category or add new category, but it is not recommended to change the primary id and name of category that already existed.
-
 ## 7.2. License
 
 YMIR is licensed under version 2.0 of the Apache License. See the [LICENSE](https://github.com/IndustryEssentials/ymir/blob/master/LICENSE) file for details.
 
 ## 7.3. Contact us
 
-Contact us with further questions：contact.viesc@gmail.com
+Contact us with further questions:contact.viesc@gmail.com
+
+Or join our SLACK, we will answer your questions in real time: https://join.slack.com/t/ymir-users/shared_invite/zt-ywephyib-ccghwp8vrd58d3u6zwtG3Q
```

Comment:
Join our [Slack community](https://join.slack.com/t/ymir-users/shared_invite/zt-ywephyib-ccghwp8vrd58d3u6zwtG3Q) to chat to our engineers about your use cases, questions, and support queries.

Reply:
`Join our [Slack community](https://join.slack.com/t/ymir-users/shared_invite/zt-ywephyib-ccghwp8vrd58d3u6zwtG3Q) to chat to our engineers about your use cases, questions, and support queries.`